src/routes.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
router.js will keep all nav related details. You might also need to connect to different components based on the route. I think no need to extract it out to MainNavPublic. Check this out: https://stackoverflow.com/questions/47338077/react-router-v4-navlink-vs-link-benefits

Reply:
I'm not sure I'm following you here. Can you please elaborate?

Reply:
I mean, you can simple place your NavLinks in the router.js file. The file is going to have routing details, and links only. So you need not place that in a separate component

Reply:
Ok. will work on it first thing tomorrow morning 😉

Reply:
@meenakshi-dhanani . just wanna give you an update. I have put back the navbar inside route.js and pushed the changes.
Can you pls re-review this PR now? thanks